### ISSUE TITLE
Issue with APC notifications for authors in OJS3

### ISSUE THREAD

**Issue by kshuttle** [pkp/pkp-lib#6248]
**Describe the bug**
When you enable author fees in OJS 3.1 (in both the Subscription and Distribution settings), there's an option to notify the author that the fee is due when the editor accepts the submission after peer review. However, this option does not appear in OJS 3.2.

This is what it looks like in 3.1 when payment is enabled and the editor is informing the author that their submission is accepted after review:

![Screenshot_2020-09-17 Samantha Jones, Community Scholars](https://user-images.githubusercontent.com/59741486/93926786-d24cd500-fccc-11ea-97a2-1ed49926ce89.png)

In 3.2, the option to add a payment is missing when the editor is notifying the author that their submission has been accepted.

![Screenshot_2020-09-22 OJS-3 2-Payment-option-missing](https://user-images.githubusercontent.com/59741486/93927028-2952aa00-fccd-11ea-96a9-c1ce152e510a.png)

In both cases, payments are enabled in the Distribution settings:

![Screenshot_2020-09-22 Distribution Settings](https://user-images.githubusercontent.com/59741486/93927189-61f28380-fccd-11ea-8f05-98ae730d17aa.png)

And under Payment Types in the Subscriptions settings:

![Screenshot_2020-09-22 Subscriptions](https://user-images.githubusercontent.com/59741486/93927234-70409f80-fccd-11ea-83b0-6f02514e71a9.png)

**To Reproduce**
Enable payments for the journal as described above, then run through a submission all the way through peer review, up to the point where the editor is notifying the author that their article has been accepted. If the journal wishes to charge an APC, they will need to notify the author and give them directions for making the payment. This option seems to be missing in OJS 3.2, so it is not clear how authors can receive a notification of and directions to pay the APC.

**What application are you using?**
 OJS 3.2

**Comment by asmecher** [comment]
@kshuttle, I think the issue is that the PayPal plugin is selected but is not fully configured. This will cause the payment tools to avoid attempting to interact with PayPal. I suspect if you add test credentials for PayPal, the APC request will re-appear.

**Comment by kshuttle** [comment]
@asmecher thanks - that worked! I also tried this with the manual payment option enabled for the journal. The option still didn't appear in the notification to authors until I added some text to the "manual fee payment instructions." It makes sense that those details are required before the option appears to send a notification to authors.
![Screenshot_2020-09-22 Distribution Settings-Manual-fee-payment-instructions](https://user-images.githubusercontent.com/59741486/93935931-45a91380-fcda-11ea-8fa4-a11946d6cb1a.png)
